kvrocks.conf

```diff
@@ -96,6 +96,15 @@ dir /tmp/kvrocks
 # We also can send logs to stdout/stderr is as simple as:
 #
 log-dir stdout
+# log-dir log
+
+# To enable the log cleaner. if set yes , glog will check if there are
+# overdue logs whenever a flush is performed
+enablelogcleaner yes
```

Comment:
I think a single `logcleanerday` item is sufficient, with a value of 0(or -1) to turn off logcleaner.

Reply:
thanks, i get it